addons/portal/data/mail_templates.xml

```diff
@@ -3,8 +3,9 @@
 
     <template id="portal_share_template">
         <div>
+            <br/>
             <p>Dear <span t-esc="partner.name"/>,</p>
-            <p>You have been invited to access the following document:</p>
+            <p><span t-out="record.env.user.name"/> has invited you to access the following <span t-out="record_description"/>:</p>
```

Comment:
Don't we have "user" in the rendering context that is the current user ?

Reply:
Indeed, you right. 🙂